dashboard/lib/tasks/asset_sync.rake

```diff
@@ -14,9 +14,26 @@ namespace :assets do
     app.assets = Sprockets::Railtie.build_environment(app)
     Sprockets::Railtie.build_manifest(app).compile('application.js')
   end
+
+  desc 'Copy digested assets to non-digested file paths'
+  task no_digests: :environment do
+    assets = Dir.glob("#{dashboard_dir}/public/assets/js/**/*")
+    regex = /(-{1}[a-z0-9]{32}*\.{1}){1}/
```

Comment:
This regex doesn't make sense to me. Isn't every `{1}` in this regex redundant?  `/-/` matches exactly one time the same as `/-{1}/`.  Also, what does the `*` after `{32}` do?

I think this could just be:
```ruby
regex = /(-[a-z0-9]{32}\.)/
```

What am I missing?